packages/extension/.gitignore

```diff
@@ -1,26 +1,24 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
-# dependencies
-/node_modules
-/.pnp
-.pnp.js
-
-# testing
-/coverage
-
-# production
```

Comment:
The production folder should be added back, if you create a build it will want to be added to the repository

Reply:
With this config, the build is generated into the `dist` directory